docs/modules/chat.mdx

```diff
@@ -5,14 +5,28 @@ title: Chat
 This addon requires you to have a matching version of EssentialsX installed.
 :::
 
+EssentialsX Chat is a lightweight chat formatter offering basic features. It allows you to customize chat formatting on
+your server by adding prefixes, suffixes and colors in chat, as well as setting different formats for different groups.
 
-EssentialsX Chat is a basic chat formatting addon for EssentialsX.
+## Configuration
+You can access the EssentialsChat configuration by opening `/plugins/Essentials/config.yml`, and scrolling down until
+you see the section marked "EssentialsX Chat".
 
+In Vanilla Minecraft, all players can see all other player messages. This setting will change the number of blocks (In
+a radius) messages can be viewed from. Set to 0 to make all chat global.
 
-## Configuration
-You can access the EssentialsChat configuration by opening `/plugins/Essentials/config.yml`, and scrolling down until you see the section marked "EssentialsX Chat"
+::::note
+Users with the `essentials.chat.spy` permission will hear everything, regardless of this setting.
```

Comment:
i don't think "hear" is the right word to use. 
i get the context, but i think "see" is better suited.

Reply:
To be addressed on next commit.